support-frontend/assets/pages/[countryGroupId]/oneTimeCheckout.tsx

```diff
@@ -197,17 +228,28 @@ function OneTimeCheckoutComponent({
 	const { amounts, defaultAmount, hideChooseYourAmount } =
 		amountsCardData['ONE_OFF'];
 
+	const { preSelectedPriceCard, preSelectedOtherAmount } = getPreSelectedAmount(
+		preSelectedAmountParam,
+		amounts,
+	);
+
 	const minAmount = config[countryGroupId]['ONE_OFF'].min;
 
-	const [amount, setAmount] = useState<number | 'other'>(defaultAmount);
-	const [otherAmount, setOtherAmount] = useState<string>('');
+	const [selectedPriceCard, setSelectedPriceCard] = useState<number | 'other'>(
```

Comment:
Great changing this naming from `amount` to `selectedPriceCard`, could we use `priceCardAmount`?

Reply:
I wanted to avoid calling it an amount, because that felt confusing when "other" is an option